ui-v2/app/styles/base/reset/base-variables.scss

```diff
@@ -0,0 +1,36 @@
+/* iris: generated by iris. If you edit this file ensure you delete this comment */
+%reset-margin {
+  margin: 0;
+}
+%reset-box {
+  margin: 0;
+  padding: 0;
+}
+%reset-typo {
```

Comment:
Yes it means typography, I'd rather not use the full word as I'm using the `typo` prefix elsewhere which I prefer and I'd rather stick to that convention, I think its fairly clear what it means? I think you understood that which confirms that for me? Similarly I'm using `decor` to refer to decoration

Reply:
In order I'd prefer

`typography-`
`type-`
`typo-`

Mostly because typo is already a word that doesn't mean typography. But those are just my $0.02.